app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectsController < ApplicationController
+
+    def show
+        @project = Project.find(params[:id])
+        @challenge = Project.find(params[:id]).challenge
```

Comment:
This works, but is a bit overkill since we know that a Project `belongs_to` a challenge... Since `belongs_to` and `has_many` act like attr_readers, we can call `@project.challenge` to get the challenge instance that this project belongs to directly. 

Try putting in a pry here and entering that code above, to see what I mean!